speakers.html

```diff
@@ -0,0 +1,121 @@
+---
+layout: speakers
+title: Speakers
+permalink: /speakers/
+---
+
+<section id="fh5co-speakers" class="speakers-page" data-section="speakers">
+    <div class="container">
+        <div class="row">
+            <div class="col-md-12 section-heading text-center">
+                <h2 class="to-animate">Keynote Speakers</h2>
+            </div>
+        </div>
+        {% for speaker in site.data.speakers.keynote %}
+        <div class="row">
+            <div class="col-md-3 col-md-offset-2">
+                <div class="fh5co-person-pic text-center to-animate">
+                    <figure><img src="{{ "/assets/images/" | prepend: site.url }}{{ speaker.pic }}" alt="Image" class="img-circle img-responsive"></figure>
+                    <h3><strong>{{ speaker.name }}</strong></h3>
+                    <span class="fh5co-position"><em>{{ speaker.position }}</em></span>
+                </div>
+            </div>
+            <div class="col-md-6">
+                <div class="fh5co-person-bio text-left to-animate">
+                    <p>{{ speaker.bio }}</p>
+                </div>
+            </div>
+            {% if forloop.last == false %}
+            <div class="col-md-12">
+                <hr class="to-animate"/>
+            </div>
+            {% endif %}
+        </div>
+        {% endfor %}
+
+        <div class="row">
+            <div class="col-md-12 section-heading breakout text-center">
+                <h2 class="to-animate">Breakout Speakers</h2>
+            </div>
+        </div>
+        {% for speaker in site.data.speakers.breakout %}
+        <div class="row">
+            <div class="col-md-3 col-md-offset-2">
+                <div class="fh5co-person-pic text-center to-animate">
+                    <figure><img src="{{ "/assets/images/" | prepend: site.url }}{{ speaker.pic }}" alt="Image" class="img-circle img-responsive"></figure>
+                    <h3><strong>{{ speaker.name }}</strong></h3>
+                    <span class="fh5co-position"><em>{{ speaker.position }}</em></span>
+                </div>
+            </div>
+            <div class="col-md-6">
+                <div class="fh5co-person-bio text-left to-animate">
+                    <p>{{ speaker.bio }}</p>
+                </div>
+            </div>
+            {% if forloop.last == false %}
+            <div class="col-md-12">
+                <hr class="to-animate"/>
+            </div>
+            {% endif %}
+        </div>
+        {% endfor %}
+	</div>
+</section>
+
+{% comment %}
```

Comment:
I've already removed it. If you're not using these comments for anything, there's no need for them to be here.